index.css

```diff
@@ -0,0 +1,9 @@
+h1 {
```

Comment:
- [ ] Kindly link your index.css files to your index.html file so your color can be added 👍🏿